{"uuid":"6de3e71b-4adc-4aff-bb7c-7c89db90a8fb","historyId":"ba1232010bb4ab3f6ea0e56680e8ee21","fullName":"org.example.tests.TestVWOLogin_02_PropertyReader_DriverManager_POM.test_negative_vwo_login","labels":[{"name":"package","value":"org.example.tests.TestVWOLogin_02_PropertyReader_DriverManager_POM"},{"name":"testClass","value":"org.example.tests.TestVWOLogin_02_PropertyReader_DriverManager_POM"},{"name":"testMethod","value":"test_negative_vwo_login"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SeleniumAdvanceFramework"},{"name":"subSuite","value":"org.example.tests.TestVWOLogin_02_PropertyReader_DriverManager_POM"},{"name":"host","value":"Yashu"},{"name":"thread","value":"17900@Yashu.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"test_negative_vwo_login","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Cannot invoke \"org.openqa.selenium.WebDriver.get(String)\" because \"this.driver\" is null","trace":"java.lang.NullPointerException: Cannot invoke \"org.openqa.selenium.WebDriver.get(String)\" because \"this.driver\" is null\r\n\tat org.example.pages.pageObjectModel.appvwo.normal_POM.LoginPage.loginwithInValidCrds(LoginPage.java:19)\r\n\tat org.example.tests.TestVWOLogin_02_PropertyReader_DriverManager_POM.test_negative_vwo_login(TestVWOLogin_02_PropertyReader_DriverManager_POM.java:27)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"TC#1- Verify that with invalid email, pass, error message is shown on the app.vwo.com","steps":[],"attachments":[],"parameters":[],"start":1746546010800,"stop":1746546010800}